velox/functions/prestosql/Arithmetic.h

```diff
@@ -495,5 +495,17 @@ struct TruncateFunction {
   }
 };
 
+template <typename T>
+struct NormalCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void
+  call(double& result, double m, double sd, double value) {
+    VELOX_USER_CHECK_GT(sd, 0, "standard deviation must be > 0");
```

Comment:
Please make this error message consistent with Presto java "standardDeviation must be > 0"